website_whatsapp/models/res_config_settings.py

```diff
@@ -24,6 +24,7 @@ def _inverse_whatsapp_enabled(self):
     whatsapp_text = fields.Char(
         related="website_id.whatsapp_text",
         readonly=False,
+        translate=True,
```

Comment:
I think this is not needed being a related.

Reply:
Fixed!